template/base/validator.py

```diff
@@ -183,7 +214,7 @@ def stop_run_thread(self):
             bt.logging.debug("Stopped")
 
     def __enter__(self):
-        self.run_in_background_thread()
+        self.run()
```

Comment:
Why not use run_in_background_thread? 
Seems it would help on some async functionality - as per the comment on the same method inside the miner

Reply:
![image](https://github.com/KMFODA/DistributedTraining/assets/38498170/ea80df80-97a9-4724-91e9-272ead6c9006)

Reply:
Yeah I thought it was the reason I was seeing specific errors last night but doesn't seem to be the case. Reverted.